src/System.Runtime.Extensions/tests/System/Convert.netcoreapp.cs

```diff
@@ -14,9 +14,9 @@ public partial class ConvertTests
         [InlineData(new byte[] { 5, 6, 7, 8 }, "BQYHCA==")]
         public void ToBase64String_Span_ProducesExpectedOutput(byte[] input, string expected)
         {
-            Assert.Equal(expected, Convert.ToBase64String(input.AsReadOnlySpan()));
-            Assert.Equal(expected, Convert.ToBase64String(input.AsReadOnlySpan(), Base64FormattingOptions.None));
-            Assert.Equal(expected, Convert.ToBase64String(input.AsReadOnlySpan(), Base64FormattingOptions.InsertLineBreaks));
+            Assert.Equal(expected, Convert.ToBase64String(input));
```

Comment:
This will end up calling the wrong overload. We want to test the span-based overload here. Same issue elsewhere.